platform/android/DISTRIBUTE.md

```diff
@@ -1,21 +1,30 @@
 # Distributing Mapbox GL Native for Android
 
-Depending on your use case, you may want to support all or just a subset of [Android ABIs](http://developer.android.com/ndk/guides/abis.html).
-This can be achieved using the different `Makefile` targets that are available.
+Depending on your use case, you may want to support all or just a subset of [Android ABIs](http://developer.android.com/ndk/guides/abis.html). This document covers building an `.aar` file from the Mapbox Android SDK and building `.so` files for specific ABIs. In normal circumstances an application developer will use [APK splits](https://developer.android.com/studio/build/configure-apk-splits.html) to optimize this at application level.
```

Comment:
@tobrun It'd make sense to link https://github.com/mapbox/mapbox-gl-native/wiki/Strategies-to-reduce-Android-APK-size here.